# /** ------激活控制器 ------ */

uint32          u4ObjectId
bool            u1Lateral
bool            u1Longitudinal